run_tests.sh

```diff
@@ -1 +1 @@
-rm -rf project/.test_* && py.test -s --ds=test_settings --verbose --cov ./unicoremc unicoremc "$@"
+rm -rf project/.test_* && py.test -s --ds=test_settings --verbose --cov ./unicoremc --cov ./organizations unicoremc organizations "$@"
```

Comment:
I think there's a way to do this in a `setup.cfg` file with a `py.test` block. May be better, ideally we should just be able to run `py.test` and have it load all the config bits from `setup.cfg` without needing to supply any extra args.